extensions/objects/templates/FilepathEditor.js

```diff
@@ -280,7 +285,8 @@ oppia.directive('filepathEditor', [
           var format = '';
           var chunks = $scope.data.metadata.uploadedFile.name.split('.');
```

Comment:
Seems like there's no need for the 'chunks' variable at all. Remove that piece of code?